src/ui_basic/panel.h

```diff
@@ -94,6 +92,15 @@ class Panel {
 		// Other panels will snap to this one.
 		pf_snap_target = 1 << 14,
 	};
+	/** The Z ordering of overlapping panels; highest value is always on top. */
+	enum class ZOrder : uint8_t {
+		kAlwaysInBackground = 0,  ///< Always in the background.
+		kDefault = 5,             ///< No special handling.
+		kPinned = 10,             ///< Pinned by the user.
+		kInfoPanel = 20,          ///< The info panel and toolbar.
+		kFullscreenWindow = 30,   ///< A fullscreen window.
+		kDropdown = 50,           ///< Dropdown lists.
```

Comment:
Just a fly-by comment. As you made it nearly doubling, maybe make it powers of 2, with `kDefault = 8`? (haven't checked whether it's passed down as `uin8_t` anywhere, but that's just enough for now then though)